README.md

```diff
@@ -108,9 +108,9 @@ To get a local/server copy up and running follow these simple example steps.
     npm run init
     ```
 
-    > You have to follow the steps in the `init` script. Put your token, bot-id, client-id, owner-id. The script will automatically create `config.json` & `database` for you. **:warning: This is one time only!**
+    > You have to follow the steps in the `init` script. Input your `token`, `client-id`, `owner-id` and `guild-id`. The script will automatically create `config.json` & `database` for you. **:warning: This is a one-time setup!**
```

Comment:
Could you adhere to config.json field values and change this section please?

Reply:
My original PR was made to correct the discrepancies between the information provided in README.md and the config.json fields.